pkg/scanners/cloud/aws/scanner_test.go

```diff
@@ -437,3 +439,98 @@ deny[res] {
 		})
 	}
 }
+
+func Test_AWSInputSelectorsWithConfigData(t *testing.T) {
+	testCases := []struct {
+		name            string
+		srcFS           fs.FS
+		dataFS          fs.FS
+		state           state.State
+		expectedResults struct {
+			totalResults int
+			summaries    []string
+		}
+	}{
+		{
+			name: "single cloud, single selector with config data",
```

Comment:
This is good. We can add another case where the value is a different type, for instance an int.

Reply:
okay